oscaralanpierce/skyrim_inventory_management_frontend
# Implement basic homepage

## Context

[**Build deployable app scaffold using new stack**](https://trello.com/c/yFori2Hb/228-build-deployable-app-scaffold-using-new-stack)

We've got a very basic Vite app deployed to Firebase, but now we want to customise it the way we need. This is the first of a few PRs to bring in public pages (non-authenticated ones) from the legacy frontend to the rewrite. This PR adds a basic homepage.

## Changes

* Create `HomePage` component with styling, tests, and stories
* Configure React Router to switch between pages (eventually, when there are multiples)
* Replace existing `App` component with new object representing our routes
* Add Google fonts to Vite as well as Storybook

## Considerations

I decided to break down implementation of this card into multiple PRs. This way, the size of each PR should stay manageable.

## Manual Test Cases

This PR introduces a static page. You should run it in your dev environment to make sure:

* The text is centred both horizontally and vertically
* The design is responsive at supported sizes (>= 425px)
* The Google font is loading correctly

Check Storybook for the font as well.

## Screenshots and GIFs

### 425px

<img width="422" alt="HomePage-425" src="https://user-images.githubusercontent.com/5115928/220059633-1a00d3df-d932-45b6-93f1-c98f2b3a1318.png">

### 601px

<img width="599" alt="HomePage-601" src="https://user-images.githubusercontent.com/5115928/220059737-8d3fb031-b7c7-4ff2-b055-cdd1a20fea3c.png">

### 769px

<img width="766" alt="HomePage-769" src="https://user-images.githubusercontent.com/5115928/220059864-d55d5d56-7dfe-43e1-8a28-03446b638582.png">

### 1025px

<img width="1024" alt="HomePage-1025" src="https://user-images.githubusercontent.com/5115928/220059968-529cd562-f961-4690-bd31-92deb774c245.png">

### 1201px

<img width="1199" alt="HomePage-1201" src="https://user-images.githubusercontent.com/5115928/220060050-3707530e-05a0-4dc3-ba96-420a7a466182.png">

### Large Desktop

<img width="957" alt="HomePage-Full" src="https://user-images.githubusercontent.com/5115928/220060435-ecb45ce2-398d-4ba5-93d4-420f36772bf2.png">
